src/Libraries/CoreNodeModels/Selection.cs

```diff
@@ -32,7 +32,23 @@ public abstract class SelectionBase<TSelection, TResult> : NodeModel
         
         #region public properties
 
-        public IEnumerable<TSelection> Selection { get { return selection; } } 
+        public IEnumerable<TSelection> Selection { get { return selection; } }
+
+
+        public override NodeInputData InputData
```

Comment:
@sm6srw this *should* support most Revit selection nodes automatically right?

Reply:
> @sm6srw this _should_ support most Revit selection nodes automatically right?

Yes, I think so.